modular_lumos/code/datums/components/crafting/recipes/recipes_misc.dm

```diff
@@ -0,0 +1,511 @@
+/////////////////
+//Large Objects//
+/////////////////
+
+/datum/crafting_recipe/showercurtain
+	name = "Shower Curtains"
+	reqs = 	list(/obj/item/stack/sheet/cloth = 2,
+				 /obj/item/stack/sheet/plastic = 2,
+				 /obj/item/stack/rods = 1)
+	result = /obj/structure/curtain
+	subcategory = CAT_FURNITURE
+	category = CAT_MISC
+
+/datum/crafting_recipe/guillotine
+	name = "Guillotine"
+	result = /obj/structure/guillotine
+	time = 150 // Building a functioning guillotine takes time
+	reqs = list(/obj/item/stack/sheet/plasteel = 3,
+		        /obj/item/stack/sheet/mineral/wood = 20,
+		        /obj/item/stack/cable_coil = 10)
+	tools = list(TOOL_SCREWDRIVER, TOOL_WRENCH, TOOL_WELDER)
+	subcategory = CAT_MISCELLANEOUS
+	category = CAT_MISC
+
+/datum/crafting_recipe/femur_breaker
+	name = "Femur Breaker"
+	result = /obj/structure/femur_breaker
+	time = 150
+	reqs = list(/obj/item/stack/sheet/metal = 20,
+		        /obj/item/stack/cable_coil = 30)
+	tools = list(TOOL_SCREWDRIVER, TOOL_WRENCH, TOOL_WELDER)
+	subcategory = CAT_MISCELLANEOUS
+	category = CAT_MISC
+
+/datum/crafting_recipe/noose //Lumos Change, Adds Noose and Cross
+	name = "Noose"
+	result = /obj/structure/chair/noose
+	reqs = list(/obj/item/stack/cable_coil = 20)
+	time = 70
+	subcategory = CAT_MISCELLANEOUS
+	category = CAT_MISC
```

Comment:
you only need this bit, no need to redefine the rest of the recipies in this file